index.html

```diff
@@ -1516,55 +1532,109 @@ <h3>Introduction</h3>
-					<li>It MUST only support [=eBraille content documents=] in the <code>spine</code>. The reading
-						system MAY reject eBraille publications with other formats in the spine or ignore their entries
-						when rendering the publication.</li>
-					<li>It MUST NOT support <a data-cite="epub-rs-33#sec-xhtml-forms">form submission</a>
-						[[epub-rs-33]].</li>
-					<li>It MUST NOT support <a data-cite="epub-rs-33#sec-svg">SVG content documents</a>. (Note that this
-						does not affect <a data-cite="epub-rs-33#sec-xhtml-svg">embedded SVG</a> [[epub-rs-33]].)</li>
-					<li>It MUST NOT support EPUB 3's <a data-cite="epub-33#sec-css-prefixed">prefixed CSS properties</a>
-						[[epub-33]].</li>
-					<li>It MUST NOT support <a data-cite="epub-rs-33#sec-scripted-content">scripting</a>
-						[[epub-rs-33]].</li>
-					<li>It is not required that eBraille publications be <a data-cite="epub-rs-33#sec-container-iri"
-							>assigned a unique URL</a> [[epub-rs-33]], as scripting is not supported, but it is still
-						recommended to domain isolate publications in case scripting is introduced in a future
-						version.</li>
-					<li>It MUST NOT support <a data-cite="epub-rs-33#epub-rs-33/#sec-rendering-control">fixed
-							layouts</a> [[epub-rs-33]].</li>
-					<li>It MUST NOT support <a data-cite="epub-rs-33#sec-epub-rs-conf-backward">backward
-							compatibility</a> or <a data-cite="epub-rs-33#sec-epub-rs-conf-forward">forward
-							compatibility</a> with other EPUB versions [[epub-rs-33]].</li>
-				</ul>
+					<p>An [=eBraille reading system=] MUST meet the <a data-cite="epub-rs-33#sec-rs-conformance"
+							>requirements for EPUB reading systems</a> [[epub-rs-33]], with the following
+						differences:</p>
 
-				<p>In addition, eBraille reading systems MUST support PDF for tactile graphics. If an eBraille reading
-					system is not capable of rendering PDFs embedded in [=eBraille content documents=], it MUST provide
-					a mechanism that allows the user to open the graphics in an application that supports PDF
-					rendering.</p>
+					<ul>
+						<li>It MUST NOT support <a data-cite="epub-rs-33#sec-epub-rs-conf-remote-res">remote
+								resources</a> [[epub-rs-33]].</li>
+						<li>It MUST NOT resolve [=path-absolute-URL strings=] [[url]].</li>
+						<li>As remote resources, forms, and scripting are not supported, it MUST NOT allow [=eBraille
+							publications=] to have <a data-cite="epub-rs-33#sec-epub-rs-network-access">network
+								access</a> [[epub-rs-33]]. (Note that barring network access should not prevent users
+							from following <a data-cite="epub-rs-33#sec-epub-rs-external-links">external hyperlinks</a>
+							[[epub-rs-33]] in a publication.)</li>
+						<li>It MUST only support [=eBraille content documents=] in the <code>spine</code>. The reading
+							system MAY reject eBraille publications with other formats in the spine or ignore their
+							entries when rendering the publication.</li>
```

Comment:
Maybe this is a lack of understanding on my part, but how can you have multiple renditions in an eBraille, e.g. braille and audio and at the same time the spine only contains braille?

Shouldn't we be less restrictive with the spine in light of multiple renditions?

Reply:
My understanding of multiple renditions is that they're wanted for providing text for tts. If you're trying to put audio in the spine, then we'd have to bring back manifest fallbacks as there's no ability to do this without including a fallback to a text document (this is true for epub, as well). If you want media overlays in an alternative rendition, you still need to have text documents in the spine to synchronize with the audio.

There's an exception in the multiple renditions section that doesn't require additional renditions to meet the braille character encoding requirements, so that case is supported. The xhtml still has to conform to the ebraille content document definition, but that shouldn't be problematic if the text structure is the same as the braille.

Reply:
After having talked to @ManfredMuchenberger I understand now. All fine.